### PR TITLE
[RM Ch06]  Fix Table 6.3

### LINKED ISSUE #1709: [RM Ch06] Fix Table 6.3 Format
Rendering of Table 6.3 has issues as md doesn't support merged cells
Also change bogometer

### PR DESCRIPTION
**Pull request by pgoyal01**
Move Table 6.3 from md to html to allow rowspans

Also changed the bogo meter to "dickering over fine points"

Fixes #1709

### CODE REVIEW

**Comment by pgoyal01** [comment]
@walterkozlowski I couldn't find Tom (GSMA) -- can we ask him to review Table 6.3. Also, do we have to make the same change in the Baldy release?

File: doc/ref_model/chapters/chapter06.md [hunk 1]
```diff
@@ -98,69 +98,63 @@ ETSI ETSI (Ref: NFV IFA 019 v03101p) has defined a set of technology independent
...
-| GetAlarmInfoRequest|GetAlarmInfoResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Input|Filter|Specifies the accelerators and related alarmsThe filter could include accelerator information, severity of the alarm, etc. |
-| |||Output|Alarm|Information about the alarms if filter matches an alarm. |
-|  |
-| AccResourcesDiscoveryRequest|AccResourcesDiscoveryResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Output|discoveredAccResourceInfo|Information on the acceleration resources discovered within the NFVI. |
-| OnloadAccImageRequest|OnloadAccImageResponse|VIM → NFVI|Input|accResourceId|Identifier of the chosen accelerator in the NFVI. |
-| |||Input|accImageInfo|Information about the acceleration image. |
-| |||Input|accImage|The binary file of acceleration image. |
+<table>
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter06.md:101]: Any particular reason for shifting to html table from markdown?

Comment (rabi-abdel) [reply doc/ref_model/chapters/chapter06.md:101]: @deebhatia HTML looks better in general and allows you to have merged cells. for complex tables with many entires, I think use plain HTML (no fancy CSS etc) should be ok. do you agree?

Comment (deebhatia) [reply doc/ref_model/chapters/chapter06.md:101]: Okay.

**Pull request merged**